Functions.cpp

```diff
@@ -0,0 +1,93 @@
+#include "Functions.h"
+#include <algorithm>
+
+using namespace std;
+
+int showMenu()
+{
+    int number;
+    cout << "Alegeti una dintre optiuni:" << endl;
+    cout << "\t 1) Adaugati animal" << endl;
+    cout << "\t 2) Afisati lista animalelor" << endl;
+    cout << "\t 3) Editati detaliile unui animal" << endl;
+    cout << "\t 4) Eliminati un animal" << endl;
+    cout << "\t 5) Iesiti din aplicatie" << endl;
+    cout << "Selectie: ";
+    cin >> number;
+    system("clear");
+    return number;
+}
+
+void addNewAnimal(vector<Animal>& animals)
+{
+    Animal a;
+    cout << "Ati ales sa adaugati un animal nou. Va rugam introduceti urmatoarele detalii: " << endl;
+    cin >> a;
+
+    animals.push_back(a);    
+}
+
+void showEntireList(vector<Animal>& animals)
+{
+    if(animals.size() == 0)
+    {
+        cout << "Niciun animal intordus!" << endl;
+        return;
+    }
+        
+    for(int i = 0; i < animals.size(); i++) 
+    {   
+        cout << i+1 << ")" << animals[i] << endl;
+    }
+        
+}
+
+
+
+void changeAnimalDetails(vector<Animal>& animals)
+{
+    Animal updated;
+    int indexToUpdate;
+
+    if(animals.size() == 0)
+    {
+        cout << "Niciun animal intordus!" << endl;
+        return;
+    }
```

Comment:
1. deja ai print-ul asta mai jos cand chemi `showEntireList`
2. daca tot faci un quick escape cu if-ul asta, fa-l inainte sa definesti variabilele, gen fix primul lucru cand intri in functie